### PR TITLE
@noarkhh/migrate arrow

### PR DESCRIPTION
**Pull request by Noarkhh**

### CODE REVIEW

File: assets/images/xp.png
Comment (wojtechb) [inline assets/images/xp.png:1]: Just use arrow PNG.

File: src/screens/NavigationArrow.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    const next_stand_id = route.params.next_stand_id
+
+    const current_stand_position = data.tags[current_stand_id].position
+    const current_stand_angle = data.tags[current_stand_id].angle
+    const next_stand_position = data.tags[next_stand_id].position
+
+    // const current_stand_position = { x: 0, y: 0 }
+    // const current_stand_angle = Math.PI * 1 / 4
+    // const next_stand_position = { x: 1, y: 1 }
+
```
Comment (wojtechb) [inline src/screens/NavigationArrow.tsx]:
```suggestion
```

File: src/screens/NavigationArrow.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                <TouchableOpacity style={styles.button} onPress={() => console.log('Change screen')}>
+                    <Text style={styles.buttonText}>Scan Code</Text>
+                </TouchableOpacity>
+            </View>
+        </View>
+    );
+}
+
+function getArrowAngle(current_stand_position: Position, current_stand_angle: number, next_stand_position: Position) {
+
```
Comment (wojtechb) [inline src/screens/NavigationArrow.tsx]:
```suggestion
```

File: src/screens/NavigationArrow.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    const vector_between_stands = {
+        x: (next_stand_position.x - current_stand_position.x),
+        y: (next_stand_position.y - current_stand_position.y)
+    }
+
+    const angle_between_vectors =
+        Math.atan2(vector_between_stands.y, vector_between_stands.x) -
+        Math.atan2(current_stand_unit_vector.y, current_stand_unit_vector.x)
+
+    return `${angle_between_vectors * 180 / Math.PI}deg`
```
Comment (wojtechb) [inline src/screens/NavigationArrow.tsx:49]: Ok, keep your secrets then

File: src/screens/NavigationArrow.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        height: 50,
+        backgroundColor: '#fff',
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+    buttonText: {
+        fontSize: 20,
+        color: '#000',
+    },
+})
```
Comment (wojtechb) [inline src/screens/NavigationArrow.tsx]:
```suggestion
})

```

File: src/types.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    MainScreen: undefined
+}
+
+export type NavigationArrowNavigationProp = StackNavigationProp<StackParams, 'NavigationArrow'>
+export type NavigationArrowRouteProp = RouteProp<StackParams, 'NavigationArrow'>
+
+export interface NavigationArrowProps {
+    navigation: NavigationArrowNavigationProp
+    route: NavigationArrowRouteProp
+}
```
Comment (wojtechb) [inline src/types.ts]:
```suggestion
}

```

**Review by wojtechb — APPROVED** [review]
🚀🚀🚀🚀🚀

File: app/index.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  title: {
+    fontSize: 64,
+    fontWeight: "bold",
+  },
+  subtitle: {
+    fontSize: 36,
+    color: "#38434D",
+  },
+});
```
Comment (Pawel-La) [inline app/index.tsx]: Why?

**Review by Pawel-La — APPROVED** [review]

**Pull request merged**